tests/functional/cli_test.py

```diff
@@ -9,3 +18,168 @@ def test_help():
 def test_version():
     """Test the slack-annotations --version command."""
     run(["slack-annotations", "--version"], check=True)
+
+
+@pytest.fixture
```

Comment:
We normally put all fixtures at the bottom of the file (or if the fixtures are inside a class, at the bottom of that class).